README.md

```diff
@@ -1,463 +1,475 @@
+# tough-cookie
+
+[RFC 6265](https://tools.ietf.org/html/rfc6265) Cookies and CookieJar for Node.js
 
 [![npm package](https://nodei.co/npm/tough-cookie.png?downloads=true&downloadRank=true&stars=true)](https://nodei.co/npm/tough-cookie/)
 
 [![Build Status](https://travis-ci.org/salesforce/tough-cookie.svg?branch=master)](https://travis-ci.org/salesforce/tough-cookie)
 
-# Synopsis
+## Synopsis
 
-``` javascript
-var tough = require('tough-cookie');
+```javascript
+var tough = require("tough-cookie");
 var Cookie = tough.Cookie;
 var cookie = Cookie.parse(header);
-cookie.value = 'somethingdifferent';
+cookie.value = "somethingdifferent";
 header = cookie.toString();
 
 var cookiejar = new tough.CookieJar();
-cookiejar.setCookie(cookie, 'http://currentdomain.example.com/path', cb);
-// ...
-cookiejar.getCookies('http://example.com/otherpath',function(err,cookies) {
-  res.headers['cookie'] = cookies.join('; ');
+// Asynchronous!
+var cookie = await cookiejar.setCookie(
+  cookie,
+  "https://currentdomain.example.com/path"
+);
+var cookies = await cookiejar.getCookies("https://example.com/otherpath");
+// Or with callbacks!
+cookiejar.setCookie(
+  cookie,
+  "https://currentdomain.example.com/path",
+  function (err, cookie) {
+    /* ... */
+  }
+);
+cookiejar.getCookies("http://example.com/otherpath", function (err, cookies) {
+  /* ... */
 });
 ```
 
-# Installation
+Why the name? NPM modules `cookie`, `cookies` and `cookiejar` were already taken.
 
-It's _so_ easy!
+## Installation
 
-`npm install tough-cookie`
+It's _so_ easy! Install with `npm` or your preferred package manager.
 
-Why the name?  NPM modules `cookie`, `cookies` and `cookiejar` were already taken.
-
-## Version Support
+```sh
+npm install tough-cookie
+```
 
-Support for versions of node.js will follow that of the [request](https://www.npmjs.com/package/request) module.
+## Node.js Version Support
 
-# API
+Inspired by [AVA's support policy](https://github.com/avajs/ava/blob/v4.0.0/docs/support-statement.md), we follow the [node.js release schedule](https://github.com/nodejs/Release#release-schedule) and support versions that are in Active LTS or Maintenance. We only drop support for older versions of node when we do a major release.
 
-## tough
+## API
 
-Functions on the module you get from `require('tough-cookie')`.  All can be used as pure functions and don't need to be "bound".
+### tough
 
-**Note**: prior to 1.0.x, several of these functions took a `strict` parameter. This has since been removed from the API as it was no longer necessary.
+The top-level exports from `require('tough-cookie')`. All can be used as pure functions and don't need to be bound.
 
-### `parseDate(string)`
+#### `parseDate(string)`
 
-Parse a cookie date string into a `Date`.  Parses according to RFC6265 Section 5.1.1, not `Date.parse()`.
+Parse a cookie date string into a `Date`. Parses according to RFC 6265 Section 5.1.1, not `Date.parse()`.
```

Comment:
include link to [Section 5.1.1](https://datatracker.ietf.org/doc/html/rfc6265#section-5.1.1) of the RFC - https://datatracker.ietf.org/doc/html/rfc6265#section-5.1.1